kalite/distributed/static/js/distributed/user/hbtemplates/navigation.handlebars

```diff
@@ -45,6 +45,9 @@
       </li>
       <li role="presentation" class="divider"></li>
     {{/unless}}
+      <li role="presentation">
+          <a role="menuitem" tabindex="-1" href="{{ url "about" }}" id="nav_about">{{_ "About" }}</a>
+      </li>
```

Comment:
There needs to be a class around this `{{#if is_admin}}` because it shouldn't be displayed to all users!